core/bin/external_node/src/main.rs

```diff
@@ -210,6 +217,40 @@ async fn init_tasks(
         }
     }));
 
+    let pruning_enabled = config.optional.pruning_data_retention_hours.is_some();
+    if pruning_enabled {
+        let l1_batch_age_to_prune =
+            Duration::from_secs(3600 * config.optional.pruning_data_retention_hours.unwrap());
```

Comment:
```suggestion
    if let Some(data_retention_hours) = config.optional.pruning_data_retention_hours {
        let l1_batch_age_to_prune = Duration::from_secs(3600 * data_retention_hours);
```
An `.unwrap()` and some copypaste can be saved here i think

Reply:
indeed, corrected